### DISCUSSION TITLE
Feature Request Solis S6

### DISCUSSION THREAD

**Discussion by whyisthisbroken** [StephanJoubert/home_assistant_solarman#79] [Q&A]
Hi,

it would be awesome it was a yaml for the Solis S6 :D

**Comment by gedger** [comment]
Have you tried the config I submitted for the Solis 8k 5g?

https://github.com/StephanJoubert/home_assistant_solarman/issues/48

**Comment by whyisthisbroken** [comment]
Hi, yes i tried - changed a little bit.
But now i sell my Datalogger.
Everytime he connects to the SolisCloud (Solarman Cloud) the Logger set my Inverter Time wrong (+6 Hours) and i dont know why.
That sucks a lot and i dont find anything or a solution in the web.

I dont know what to do with this thing anymore. :(
